boxes/membrane/packer.json

```diff
@@ -70,15 +71,67 @@
     }
   ],
   "post-processors": [
+    [
+      {
+        "keep_input_artifact": true,
+        "only": [
+          "{{ user `virtualbox_identifier` }}"
+        ],
+        "output": "builds/{{ user `group_id` }}/{{ user `artifact_id` }}/{{ user `major_version` }}.{{ user `minor_version` }}.{{ user `patch_version` }}.{{ user `incremental_version` }}/vagrant/{{ user `group_id` }}.{{ user `virtualbox_identifier` }}.{{ user `artifact_id` }}-{{ user `major_version` }}.{{ user `minor_version` }}.{{ user `patch_version` }}.{{ user `incremental_version` }}.box",
+        "type": "vagrant"
+      },
+      {
+        "acl": "public-read",
+        "box_dir": "boxes",
+        "box_name": "{{ user `group_id` }}.{{ user `aws_identifier` }}.{{ user `artifact_id` }}",
+        "bucket": "{{ user `aws_vagrant_bucket_name` }}",
+        "manifest": "{{ user `group_id` }}.{{ user `aws_identifier` }}.{{ user `artifact_id` }}",
+        "only": [
+          "{{ user `virtualbox_identifier` }}"
+        ],
+        "region": "{{ user `aws_region` }}",
+        "type": "vagrant-s3",
+        "version": "{{ user `major_version` }}.{{ user `minor_version` }}.{{ user `patch_version` }}.{{ user `incremental_version` }}"
+      }
+    ],
     {
-      "keep_input_artifact": true,
+      "environment_vars": [
+        "OUTPUT_FILE=builds/{{ user `group_id` }}/{{ user `artifact_id` }}/{{ user `major_version` }}.{{ user `minor_version` }}.{{ user `patch_version` }}.{{ user `incremental_version` }}/variables.json",
+        "username=vagrant",
```

Comment:
Good point. Ignore this. I don't think there is a user data equivalent for Vagrant